onboarding/mentors.md

```diff
@@ -6,23 +6,63 @@ description: How to be an effective mentor of a new hire
 # Mentorship at Artsy
 
 ## Why Mentor
-One of Artsy's core values is that [People are Paramount](/culture/what-is-artsy.md#people-are-paramount). As a company we try to meet this in a number of ways, from all the work of People Ops to Ongoing development conversations between employees and managers. As an engineering team we also try to reflect this value by encouraging mentoring among peers.
 
-What does this look like in practice? In an ideal world it would be automatic and spontaneous between every member of the team. Some of our [engineering principles](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#readme) appeal explicitly to this ideal - **Psychological Safety**, **Leverage Your Impact**, **De-silo Engineers** and **Being Nice is Nice** stand out - while others like **Open Source by Default** and **Incremental Revolution**, while technical on the surface have clear analogues in how we work together. In practice we understand that all of these ideals require intentional work and iteration between people to make them reality. Artsy mentors, assigned when a new engineer joins the team, work to bridge that gap.
+We are proud of our
+[engineering principles](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#readme) and
+team culture, from its commitment to excellence in software development to its impact on the wider software
+engineering community. Of course, no culture is static; rather than a one-time goal it is something that takes work
+to curate, maintain and protect. We recognize that all of these ideals require intentional work and iteration
+between people to make them reality. One way we work to maintain and grow this culture is through mentorship.
 
-## What makes a good mentor
-The role of a mentor is purposely different from that of a manager or technical lead, and that role shifts to accommodate the needs of a team member. There may be multiple overlapping descriptions - a mentor may be someone who
-- *Facilitates onboarding*: Simply someone who can help a new hire smooth out the bumps of getting onboarded.
-- *Knows how things work*: From a non-judgmental sounding board for [<strike>stupid</strike> questions](#simple-answers-to-common-questions) to someone who can simply help find the best point person or stakeholder for a project.
-- *Facilitates career development*: From apprenticeship to pointing someone in the direction of an article, project or pair where they can learn something new, mentoring pairs can help uncover or set new goals and offer help in meeting them.
+Although mentorship can occur in many forms, currently on the Artsy engineering team our only formal mentorship
+program is for new hires. Although both the company and engineering team have programs that fall under the umbrella
+of _mentorship, career development_ and _education_, this doc will focus on mentorship in the context of onboarding
+to the engineering team.
```

Comment:
👍

Reply:
This bit sounds apologetic, when I think you're actually going for clarity. You might say "Mentorship and professional development occur in many forms and contexts, but this doc is focused on mentoring for new engineering hires."